app/models/project/copy.rb

```diff
@@ -142,7 +142,7 @@ def copy_work_packages(project)
         # Reassign fixed_versions by name, since names are unique per
         # project and the versions for self are not yet saved
         if issue.fixed_version
-          new_version = versions.to_a.detect { |v| v.name == issue.fixed_version.name }
+          new_version = versions.detect { |v| v.name == issue.fixed_version.name }
```

Comment:
@marutosi I didn't know this would work (`AR::Associations::CollectionProxy` neither implements `#detect` nor is `Enumerable`)